task-13/src/main/resources/db/changelog/v-1.0/01-create-usr-table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE usr (
```

Comment:
if we use liquibase we shouldn't have sql statements. we should describe our ;ogic in liquibase language to support different DBMS